### PR TITLE
INTG-528 Updated README to include response sets

### PR DESCRIPTION
**Pull request by mrinalirao**
Updated README to include response sets

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -1958,3 +1958,519 @@ Status code     | Description
...
+Parameter          | Description
+---------          | -----------
+`name`             | The name of the response set
+`responses`        | The array of responses to initialise the response set with
+
+The responses should have the following properties:
+
+Responses          | Description
+---------          | -----------
+`label`            | The label of the response
```
Comment (dimitrikoubaroulis) [inline source/index.html.md]: The text of the response

File: source/index.html.md [hunk 2]
```diff
@@ -1958,3 +1958,519 @@ Status code     | Description
...
+`colour`           | Optional parameter to specify the colour 
+`enable_score`     | Optional Parameter for enabling scoring in response object
+`score`            | Optional Parameter for specifying the score associated with the response which defaults to 1
+`failed`           | Optional Parameter for marking responses as `failed` for reporting analytics
+`short_label`      | Optional Parameter for specifying short label of the response
```
Comment (dimitrikoubaroulis) [inline source/index.html.md]: a short label (where is this shown?)

File: source/index.html.md [hunk 3]
```diff
@@ -1958,3 +1958,519 @@ Status code     | Description
...
+       "id": "64285441-782f-4891-b0f7-2beffbdd78bc",
+       "label": "John Smith",
+       "score": 1,
+       "short_label": "John"
+     }
+   ],
+   "created_at": "2017-07-21T03:35:26.323Z",
+   "modified_at": "2017-07-21T03:35:26.323Z"
+}
+
```
Comment (dimitrikoubaroulis) [inline source/index.html.md]: empty lines needed?

File: source/index.html.md [hunk 4]
```diff
@@ -1958,3 +1958,519 @@ Status code     | Description
...
+curl “ https://api.safetyculture.io/response_sets " \
+-H “Authorization: Bearer …” \
+-H "Content-Type:application/json"
+```
+> The above command returns an array of JSON objects structured like this
+
+```json
+[
+  {
+    "name": "employee1",
```
Comment (dimitrikoubaroulis) [inline source/index.html.md]: For public doco I suggest using friendlier names like John Smith, Joe Bloggs

File: source/index.html.md [hunk 5]
```diff
@@ -1958,3 +1958,519 @@ Status code     | Description
...
+{
+  "colour": "250,250,250",
+  "enable_score": true,
+  "failed": false,
+  "id": "64285441-782f-4891-b0f7-2beffbdd78bc",
+  "label": "New label",
+  "score": 1,
+  "short_label": "New"
+}
+
```
Comment (dimitrikoubaroulis) [inline source/index.html.md]: Empty lines in a number of cases e.g. here too

File: source/index.html.md [hunk 6]
```diff
@@ -1958,3 +1958,519 @@ Status code     | Description
...
+
+```
+
+### HTTP Request
+
+`PUT https://api.safetyculture.io/response_sets/<responseset_id>/responses/<response_id>/`
```
Comment (dimitrikoubaroulis) [inline source/index.html.md]: Do you need the extra / at the end? I suggest remove for consistency

File: source/index.html.md [hunk 7]
```diff
@@ -1958,3 +1958,519 @@ Status code     | Description
...
+
+### URL Parameters
+
+Parameter         | Description
+---------         | -----------
+`responseset_id`  | The ID of the response set which contains the response to be updated
+`response_id`     | The ID of the response object to update
```
Comment (dimitrikoubaroulis) [inline source/index.html.md]: response object -> response (everywhere, unless needed)

File: source/index.html.md [hunk 8]
```diff
@@ -1958,3 +1958,519 @@ Status code     | Description
 
 See the [Errors](#error-codes) section for more details.
 
+# Response Sets
+
+A response set is a collection of responses. Endpoints are provided to manipulate response sets as well as individual responses within them.
```
Comment (mrinalirao) [inline source/index.html.md]: nope, order does not matter here

File: source/index.html.md (hunk 2 above)
Comment (mrinalirao) [inline source/index.html.md]: The user has an option of specifying a short label, its optional. If he chooses not to specify, then it will not be stored

**Pull request merged**